src/mixins/Syncable.ts

```diff
@@ -147,15 +151,18 @@ const SyncableMixin = types
       const notSuppressed = self.syncManager!.sync(data, event, self.name);
 
       if (notSuppressed && event === 'play') {
-        self.syncMuted(false);
+        // if that's audio, it should not be muted.
```

Comment:
It may be it's better to bring the cause from the commit description here for providing the context of the problem.